src/Entity/OgRoleInterface.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal\og\Entity\OgRoleInterface.
+ */
+namespace Drupal\og\Entity;
+
+use Drupal\Core\Config\Entity\ConfigEntityInterface;
+
+/**
+ * Provides an interface defining an OG user role entity.
+ */
+interface OgRoleInterface extends ConfigEntityInterface {
```

Comment:
this class probably need only the `const` and nothing else, as we already extend from `Role`